packages/x-data-grid/src/hooks/features/export/useGridPrintExport.tsx

```diff
@@ -128,9 +131,9 @@ export const useGridPrintExport = (
       const normalizeOptions = {
         copyStyles: true,
         hideToolbar: false,
-        hideFooter: false,
         includeCheckboxes: false,
         ...options,
+        hideFooter: props.hideFooter || options?.hideFooter,
```

Comment:
Shouldn't it be something like `options.hideFooter ?? props.hideFooter`?

Reply:
I think we can do it without depending on `props.hideFooter` value ? see [27e94d7](https://github.com/mui/mui-x/pull/14863/commits/27e94d70435073b3b6c170e8d7a41453542ee0f7)